js/src/setup-ads/setup-ads-form.js

```diff
@@ -27,9 +29,11 @@ const SetupAdsForm = () => {
 	const [ handleSetupComplete, isSubmitting ] = useAdsSetupCompleteCallback();
 	const adminUrl = useAdminUrl();
 	const { data: countryCodes } = useTargetAudienceFinalCountryCodes();
+	const { highestDailyBudget, hasFinishedResolution } =
+		useFetchBudgetRecommendation( countryCodes );
 
 	const initialValues = {
-		amount: 0,
+		amount: highestDailyBudget,
```

Comment:
The default budget value may be incorrectly set due to the different currencies between WooCommerce store settings and the existing Google Ads account.

Steps to reproduce:

1. Have an existing Google Ads account with *A* currency, e.g. USD.
2. Disconnect Google Ads account if it's connected.
3. Set WooCommerce store settings to use *B* currency, e.g. JPY.
4. Go to the Ads setup flow (post-onboarding)
   1. Connect to the Google Ads account mentioned in step 1.
      ![1](https://github.com/user-attachments/assets/ffa8de35-a2a3-4bbc-94fd-caedf82cda26)
   2. Continue to step 2 and view the inconsistent currencies and incorrect initial budget.
      ![2](https://github.com/user-attachments/assets/d0b3cc67-1267-4820-b523-6045e2ae0134)

Reply:
Thanks for catching that @eason9487 . We've moved the logic to fetch the budget recommendation to `BudgetSection` now and should solve the issue.